modules/afprog/afprog.c

```diff
@@ -402,11 +402,23 @@ afprogram_dd_exit(pid_t pid, int status, gpointer s)
    * handling restarting the command before this handler is run. */
   if (self->process_info.pid != -1 && self->process_info.pid == pid)
     {
-      msg_verbose("Child program exited, restarting",
-                  evt_tag_str("cmdline", self->process_info.cmdline->str),
-                  evt_tag_int("status", status));
-      self->process_info.pid = -1;
-      afprogram_dd_reopen(self);
+      if ((status >> 8) == 127 && ((status & 0x00ff) == 0))
```

Comment:
Please use constant instead of magic numbers and/or move it to a separate function for readability.

Something like that:
```c
gboolean afprogram_dd_is_command_not_found(int status)
{
     return (status >> 8) == 127 && ((status & 0x00ff) == 0));
}
```
Some reference either in the commit message or here as a comment would be nice to the `0x00ff` and the `8` shift.